review/tests/test_review_manager.py

```diff
@@ -39,25 +39,25 @@ def test_sort_review_by_lowest_rating(self, demo_client, demo_client2, professio
                                           create_review: Callable[[str, str, str, str, int], Review]):
         self.create_reviews(demo_client, demo_client2, professional, create_review)
```

Comment:
same as above

Reply:
> same as above

Resolved (i renamed the file though git thinks i deleted it and created a new one instead)